### ISSUE TITLE
Duplicate KEEP delimiter when overwriting

### ISSUE THREAD

**Issue by olivr70** [Superschnizel/Obsidian-Moviegrabber#10]
When overwriting a movie note, the delimiter is repeated  (v1.1.5)

Exemple after overwriting %%==MOVIEGRABBER_KEEP==%% is repeated (was not before renewal)
```
## Mes notes
%%==MOVIEGRABBER_KEEP==%%
%%==MOVIEGRABBER_KEEP==%%

mes commentaires
```

**Comment by Superschnizel** [comment]
mmhm, i can not replicate this. Do you have the delimiter in your Template?

**Comment by ces3001** [comment]
I have the same issue, and yes I have the delimiter in the template.

**Comment by Superschnizel** [comment]
I have fixed this issue in the newest version 1.1.18. If there is a delimiter in the file that is being overwritten and there is a delimiter in the template, then everything below the delimiter in the template will be overwrtitten with what is in the existing file.
